addons/mail/static/src/components/chatter/chatter.js

```diff
@@ -77,6 +76,21 @@ class Chatter extends Component {
         });
     }
 
+    /**
+     * @private
+     */
+    _update() {
+        if (this.chatter.thread) {
+            this._notifyRendered();
+        }
+        if (this.chatter.isDoFocus) {
+            this.chatter.update({ isDoFocus: false });
+            const composer = this._composerRef.comp;
+            if (composer) {
+                composer.focus();
+            }
+        }
+    }
```

Comment:
empty line between last method of section and following section separator

Reply:
ok